src/uct/ib/mlx5/ib_mlx5.inl

```diff
@@ -175,6 +175,42 @@ uct_ib_mlx5_inline_copy(void *restrict dest, const void *restrict src, unsigned
 }
 
 
+/**
+ * Copy uct_iov_t array to inline segment, taking into account QP wrap-around.
+ *
+ * @param dest     Inline data in the WQE to copy to.
+ * @param iov      A pointer to an array of uct_iov_t elements.
+ * @param iov_cnt  A number of elements in a iov array.
+ * @param length   Total data length.
+ * @param wq       Send work-queue
+ *
+ */
+static UCS_F_ALWAYS_INLINE void
+uct_ib_mlx5_inline_iov_copy(void *restrict dest, const uct_iov_t *iov, size_t iovcnt,
+                            size_t length, uct_ib_mlx5_txwq_t *wq)
+{
+    size_t iov_it;
+    ptrdiff_t n;
+
+    ucs_assert(dest != NULL);
+    ucs_assert((iov != NULL) || (iovcnt == 0));
```

Comment:
I think we should make sure that `iovcnt != 0` in the function.
it doesn't work if `iovcnt == 0`

Reply:
Could you please to clarify this comment. Which function do you mean in the first sentence?

Reply:
I mean that we should expect that `iovcnt != 0` and `iov != NULL` in the `uct_ib_mlx5_inline_iov_copy()`.
We check `iovcnt > 0` in each place before calling `uct_ib_mlx5_inline_iov_copy()`

Reply:
Got it. Thanks.